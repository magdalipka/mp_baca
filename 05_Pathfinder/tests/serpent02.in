10 10
0 1 0 0 0 1 0 0 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 0 0 1 0 0 0 1 0 0
1
i 9 0 0 9

