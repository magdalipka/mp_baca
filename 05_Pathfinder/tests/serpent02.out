i W N N N N N N N N N W W S S S S S S S S S W W N N N N N N N N N W W S S S S S S S S S W W N N N N N N N N N
